pkg/kubelet/network/cni/cni.go

```diff
@@ -238,11 +213,12 @@ func (plugin *cniNetworkPlugin) SetUpPod(namespace string, name string, id kubec
 	if err != nil {
 		return fmt.Errorf("CNI failed to retrieve network namespace path: %v", err)
 	}
-
-	_, err = plugin.addToNetwork(plugin.loNetwork, name, namespace, id, netnsPath)
-	if err != nil {
-		glog.Errorf("Error while adding to cni lo network: %s", err)
-		return err
+	if plugin.loNetwork != nil {
```

Comment:
nit: add a comment to say linux comes with loNetwork and windows doesn't.

Reply:
Taken care of